python/ray/serve/_private/replica.py

```diff
@@ -1636,9 +1637,7 @@ async def _call_http_entrypoint(
 
         receive_task = None
         try:
-            receive_task = self._user_code_event_loop.create_task(
-                receive.fetch_until_disconnect()
-            )
+            receive_task = asyncio.create_task(receive.fetch_until_disconnect())
```

Comment:
can you explain why we are now able to use the system event loop to receive message ?

Reply:
we aren't, `_call_http_entrypoint` is run on the user event loop so this will create a task in the user event loop

Reply:
got it, i see that create_task uses the loop running in current thread automatically so we dont need to be explicit here https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task